WooCommerce/Classes/ViewRelated/Products/Edit Product/Product Settings/ProductSettingsViewModel.swift

```diff
@@ -4,28 +4,50 @@ import Yosemite
 /// The Product Settings contains 2 sections: Publish Settings and More Options
 final class ProductSettingsViewModel {
 
-    private(set) var sections: [ProductSettingsSectionMediator] {
-        didSet {
-            self.onReload?()
-        }
-    }
+    private let product: Product
+
+    /// The original password, the one fetched from site post API
+    private var password: String?
```

Comment:
I use the password field here because I need to check if Product Settings (which will be updated) has unsaved changes. If I use only `password` in `productSettings` I don't have any way to compare the old password with the new one.